Automattic/wp-calypso
# Marketplace: Add products to the cart from billing intent

Related to https://github.com/Automattic/dotcom-forge/issues/4939

## Proposed Changes
 
* Adds a way to fetch billing intent info from the backend
* Adds a way to add products to the cart by fetching the billing intent
* Updates the path param name for marketplace siteless checkout

## Testing Instructions



* Make sure you are logged out from WordPress.com. And you use new email for signing up for the account. 
* Visit `/checkout/marketplace/443` and make sure Passport monthly is added to the cart. 
![CleanShot 2024-01-04 at 17 01 26@2x](https://github.com/Automattic/wp-calypso/assets/86406124/3e81f129-a3ed-47df-96d4-414d3cc2b86c)

* **Make sure you don't checkout**. 

## Pre-merge Checklist
